### PR TITLE
Added phpstan validations

### PR DESCRIPTION
**Pull request by tomaj**

### CODE REVIEW

File: .github/workflows/phpstan.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: phpstan
+
+on: [push, pull_request]
+
+jobs:
+  phpcs:
+      name: PHPCS
```
Comment (lulco) [inline .github/workflows/phpstan.yml:7]: Copy paste :)

Comment (tomaj) [reply .github/workflows/phpstan.yml]: good catch, thanks! fixed

**Comment by tomaj** [comment]
after #40 we can easily increase phpstan level ;-)

**Comment by lulco** [comment]
Level 8 is new default :)

File: .github/workflows/phpstan.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+  phpcs:
+      name: phpstan
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout code
+          uses: actions/checkout@v2
+
+        - name: Composer update
+          run: composer install --dev --prefer-dist --no-progress --no-suggest
```
Comment (lulco) [inline .github/workflows/phpstan.yml:14]: You are using the deprecated option "dev". Dev packages are installed by default now.

**Pull request merged**